.github/workflows/snapshot-mapping-data-change.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'snapshot-mapping-data-change'
+
+on:
+  push:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+
+concurrency: '${{ github.workflow }}-${{ github.head_ref || github.ref }}-snapshot-mapping-data-change'
+
+jobs:
+  snapshot:
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    uses: 'abcxyz/pmap/.github/workflows/snapshot-file-change.yml@v0.0.5-rc' # Change here to pin the tag or commit.
+    with:
+      workload_identity_provider: '${{ vars.workload_identity_provider }}'
+      service_account: '${{ vars.service_account }}'
+      destination_prefix: '${{ vars.gcs_bucket }}/mapping'
+      path: 'example_org/mapping' #Change here if you change the folder to store the mappings.
```

Comment:
Should we make this repo variable?

Reply:
I think we don't need it as users may need to change the path much more frequent compared to the infra setting as wif or wif sa or gcs bucket.

Reply:
I am thinking differently, it feels like a repo set up, and should not be changed frequently.

Reply:
In addition, making it a env variable here makes it more difficult for users to understand what the path is for and users don't have the permissions to see the env variables we set in our github repo.

Reply:
discussed offline. It is fine to keep it as it is now.